docs/operations/configured-graph-factory.md

```diff
@@ -251,6 +251,10 @@ map.put("storage.hostname", "10.0.0.1");
 ConfiguredGraphFactory.updateConfiguration("graph1",
 map);
 
+// We can verify the configuration are updated by
+// retrieving configurations for this graph
```

Comment:
We can verify the configuration _are_ updated -> We can verify the configuration was updated ?
retrieving _configurations_ for this graph -> retrieving configuration?